entrypoints/services/AppStateService.ts

```
@@ -0,0 +1,70 @@
+    /**
+     * Create or update the app state
+     */
+    static async saveState(
+        state: Partial<Omit<AppState, "schemaVersion">>
+    ): Promise<AppState> {
+        try {
+            const stateRef = doc(db, this.COLLECTION, this.DOCUMENT_ID);
+            const currentState = await this.getCurrent();
+
+            const newState: AppState = {
+                schemaVersion: this.SCHEMA_VERSION,
```

Biome (1.9.4)

[error] This property value named schemaVersion is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named schemaVersion


(lint/suspicious/noDuplicateObjectKeys)